driver-core/src/main/com/mongodb/operation/CommandOperationHelper.java

```diff
@@ -105,47 +107,108 @@ public BsonDocument apply(final BsonDocument command) {
 
     /* Read Binding Helpers */
 
-    static BsonDocument executeWrappedCommandProtocol(final ReadBinding binding, final String database, final BsonDocument command) {
-        return executeWrappedCommandProtocol(binding, database, command, new BsonDocumentCodec());
+    static BsonDocument executeCommand(final ReadBinding binding, final boolean retryReads, final String database,
+                                       final BsonDocument command) {
+        return executeCommand(binding, retryReads, database, command, new BsonDocumentCodec());
     }
 
-    static <T> T executeWrappedCommandProtocol(final ReadBinding binding, final String database, final BsonDocument command,
-                                               final CommandTransformer<BsonDocument, T> transformer) {
-        return executeWrappedCommandProtocol(binding, database, command, new BsonDocumentCodec(), transformer);
+    static <T> T executeCommand(final ReadBinding binding, final boolean retryReads, final String database, final BsonDocument command,
+                                final CommandTransformer<BsonDocument, T> transformer) {
+        return executeCommand(binding, retryReads, database, command, new BsonDocumentCodec(), transformer);
     }
 
-    static <T> T executeWrappedCommandProtocol(final ReadBinding binding, final String database, final BsonDocument command,
-                                               final Decoder<T> decoder) {
-        return executeWrappedCommandProtocol(binding, database, command, decoder, new IdentityTransformer<T>());
+    static <T> T executeCommand(final ReadBinding binding, final boolean retryReads, final String database, final BsonDocument command,
+                                final Decoder<T> decoder) {
+        return executeCommand(binding, retryReads, database, command, decoder, new IdentityTransformer<T>());
     }
 
-    static <D, T> T executeWrappedCommandProtocol(final ReadBinding binding, final String database, final BsonDocument command,
-                                                  final Decoder<D> decoder, final CommandTransformer<D, T> transformer) {
+    static <D, T> T executeCommand(final ReadBinding binding, final boolean retryReads, final String database, final BsonDocument command,
```

Comment:
Can this be DRY'd up with the overloaded method on line 176 that has the same parameters plus a Connection?

Reply:
I have DRY'd up the method.